CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.11)
 project(aliceVision LANGUAGES C CXX)
-
+set(GFLAGS_USE_TARGET_NAMESPACE TRUE)
```

Comment:
Not related to the PR.
Need a comment to explain this reason for this flag.

Reply:
Junk code.